rest_framework/fields.py

```diff
@@ -936,6 +941,11 @@ def to_internal_value(self, value):
         self.fail('invalid', format=humanized_format)
 
     def to_representation(self, value):
+        if timezone.is_aware(value):
+            # convert the datetime to the timezone the user is expecting
```

Comment:
It's unclear to me that this *is* always the desired behavior. Many APIs will probably want datetimes to unconditionally be represented in UTC.